stories/LocationAutocomplete.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { text, withKnobs } from '@storybook/addon-knobs';
+import { LocationAutocomplete, LocationAutocompleteRenderer } from '@textkernel/oneui';
+
+storiesOf('Organisms|LocationAutocomplete', module)
+    .addDecorator(withKnobs)
+    .add(
+        'LocationAutocomplete',
+        () => {
+            const onSelectionChange = value => {
+                console.log('onSelectionChange was called with:');
+                console.log(value);
+            };
+            return (
+                <LocationAutocomplete
+                    apiKey=""
+                    inputPlaceholder={text('Input field placeholder', 'Enter a city or region...')}
```

Comment:
I see to the ways how we can dynamically insert  _apiKey_ .
1) Use **window.prompt()**. This method block JS flow so google doesn't init without the key.
2) Add into **additionalGoogleProps** id with apiKey, when id changes cause reinit google module